_includes/nav-top.html

```diff
@@ -9,13 +9,15 @@ <h1 class="mt2 mb0 h3 red caps ls-5">
           Identity Playbook
         </h1>
       </div>
-      <div class="sm-col-right">
-        <div class="sm-show pb-nav--top">
+      <div class="sm-col-right mxn2 sm-mx0">
+        <ul class="list-reset mt3 mb0 bg-red center pb-nav--top">
           {% assign url = page.url|remove:'index.html' %}
           {% for link in site.navigation %}
-            <a class="{% if forloop.last != true %}mr3{% endif %} gray {% if page.url == link.url %}active{% endif %}" href="{{ link.url | prepend: site.baseurl }}">{{ link.text }}</a>
+            <li class="inline-block {% if forloop.last != true %}border-right sm-mr3{% endif %}">
```

Comment:
should have `border-white` class maybe after `border-right` (if we want those to be white)